keyboards/koolertron/amag09/amag09.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2023
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+// clang-format off
+#define LAYOUT_all( \
```

Comment:
If you've only got one layout, you can just use `LAYOUT`.

```suggestion
#define LAYOUT( \
```
By the way, you probably want to move this layout/matrix info into `info.json` and then delete this file. This is one of those things that works now, but will eventually break. [You can see other examples of how this works in the repository](https://github.com/vial-kb/vial-qmk/blob/47721416a6483a1d759c21a18beae3c0cb7d3eb2/keyboards/jacky_studio/piggy60/info.json), and/or run `qmk info -f json -kb koolertron/amag09` for an idea of the syntax (though that command ends up with some duplicate data)

Reply:
Done.